exercises/practice/triangle/.meta/example.f90

```diff
@@ -25,56 +25,56 @@ logical function intol(a,b)
     intol = abs(a-b)<TOL
   end function
 
-  logical function istrianle_real(tria)
+  logical function is_trianle_real(traingle_edges)
```

Comment:
Missing a `g` in the function name, and transposed vowels.

```suggestion
  logical function is_triangle_real(triangle_edges)
```

Reply:
fixed in 54e5349